frontend/components/draft/lesson/blocknote-editor.tsx

```
@@ -0,0 +1,37 @@
+"use client";
+
+import dynamic from "next/dynamic";
+import "@blocknote/core/fonts/inter.css";
+import "@blocknote/mantine/style.css";
+
+interface BlockNoteEditorProps {
+  initialContent?: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)